components/apimgt/org.wso2.carbon.apimgt.rest.api.core/src/main/java/org/wso2/carbon/apimgt/rest/api/core/impl/ApisApiServiceImpl.java

```diff
@@ -65,6 +65,7 @@ public Response apisApiIdGatewayConfigGet(String apiId, String accept,
     }
 
     /**
+     *Retrieve a list of APIs with given gateway labels and status
```

Comment:
Thanks for pointing out the mistake. This is fixed by https://github.com/wso2/carbon-apimgt/pull/4233/commits/751208cacf54faeaa4f6e01ab3e74456faecd4c8 and https://github.com/wso2/carbon-apimgt/pull/4234/commits/9212f98d36ad3405d06db53edf79880e04108348